website/docs/components/app-footer/partials/code/component-api.md

```diff
@@ -18,7 +18,7 @@ The base `AppFooter` component includes a copyright notice. It also wraps and co
     The `AppFooter::Link` component (see below).
   </C.Property>
   <C.Property @name="<[AF].Item>" @type="yielded component">
-    The generic `AppFooter::Item` component (see below).
+    The `AppFooter::Item` component which is used for custom non-link content (see below).
```

Comment:
Since we discussed this in Slack, I decided to update this to replace the "generic" reference.